raiden/messages.py

```diff
@@ -1831,6 +1831,12 @@ def verify_request_monitoring(
             and recover(reward_proof_data, self.reward_proof_signature) == requesting_address
         )
 
+class RequestRegisterSecret(Message):
+    def to_dict(self):
+        return {
+            "type": self.__class__.__name__
+        }
+
```

Comment:
- you need 1 extra newline before this declaration
- the body for this function seems suspiciously short. could you please explain why it is coded like this, or why we need it in the first place?

Reply:
documenting what i discussed with @mauricioirace in person: for now, we only need the type of this message to know what to do with it.

the practice of coding a custom `to_dict` function is questionable, but standard for all messages so far, so we've resolved to go with what we currently have.